### PR TITLE
fix docker_start without using jq

### PR DESCRIPTION
**Pull request by AlteredCoder**

### CODE REVIEW

File: docker/docker_start.sh [hunk 1]
```diff
@@ -133,7 +133,7 @@ if [ "$DISABLE_POSTOVERFLOWS" != "" ]; then
 fi
 
 function register_bouncer {
-  if ! cscli -c "$CS_CONFIG_FILE" bouncers list -o json | jq -r .[].name | grep -q "${NAME}"; then
+  if ! cscli -c "$CS_CONFIG_FILE" bouncers list -o raw | cut -d "," -f1 | grep -q "${NAME}"; then
```
Comment (mmetc) [inline docker/docker_start.sh:136]: If you want to support names with comma or space:
cscli bouncers list -o json | sed '/^ *"name"/!d;s/^ *"name": "\(.*\)",/\1/'

then apply grep -q
tested with busybox too

File: docker/docker_start.sh [hunk 2]
```diff
@@ -133,7 +133,7 @@ if [ "$DISABLE_POSTOVERFLOWS" != "" ]; then
 fi
 
 function register_bouncer {
-  if ! cscli -c "$CS_CONFIG_FILE" bouncers list -o json | jq -r .[].name | grep -q "${NAME}"; then
+  if ! cscli bouncers list -o json | sed '/^ *"name"/!d;s/^ *"name": "\(.*\)",/\1/' | grep -q "${NAME}"; then
```
Comment (LaurenceJJones) [inline docker/docker_start.sh:136]: -c "$CS_CONFIG_FILE" no longer need this?

**Comment by mmetc** [comment]
I pushed a commit because you could have two bouncers named "foo" and "foo-something", you don't want a partial match.
Otherwise lgtm

**Review by mmetc — APPROVED** [review]

**Review by LaurenceJJones — APPROVED** [review]
lgtm

**Pull request merged**